client/src/components/commonComponents/navBar/index.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
```

Comment:
You can replace the multiple imports with one import from _'@material-ui/core'_

`import { List, ListItem, Typography, Grid, ... } from '@material-ui/core';`